test/fixtures/hello-cloud/bare_replica_set.yml

```diff
@@ -0,0 +1,27 @@
+---
+apiVersion: extensions/v1beta1
+kind: ReplicaSet
```

Comment:
Is there a use case when we would have a manifest for ReplicaSet? I thought it's created automatically by Deployment